user_front/components/Container.vue

```diff
@@ -0,0 +1,31 @@
+<script setup>
```

Comment:
```suggestion
<script lang="ts" setup>
```
typesciptなので、追加した方がよさそうです。